service/controller/machinepoolexp/handler/migration/testdata/case-0-Simple-MachinePool_MachinePool.golden

```diff
@@ -0,0 +1,60 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Expected MachinePool CR after migration handler has initially created it, with Metadata and Spec set.